Sources/LLVMWrappers/include/LLVMWrappers.h

```diff
@@ -0,0 +1,16 @@
+#ifndef LLVMWrappers_h
+#define LLVMWrappers_h
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+void *_Nullable LLVMCreateOrcMCJITReplacement(void *_Nonnull module,
```

Comment:
I don't see anything wrong. The second param is on the next line and GitHub for some reason isn't  showing the leading whitespace.